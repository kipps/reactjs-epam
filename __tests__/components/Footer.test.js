import { shallow } from 'enzyme/build';
import { configure } from 'enzyme/build';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16/build';
import Footer from "../../src/components/layout/footer";

configure({ adapter: new Adapter() });


describe("Footer ", () => {
  it("renders default footer", () =>
    expect(
      shallow(<Footer />)
        .find('footer.Footer')
        .length
    ).toBe(1));

  it('renders footer correctly', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});