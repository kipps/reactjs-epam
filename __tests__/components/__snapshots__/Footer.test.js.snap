// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer  renders footer correctly 1`] = `
<footer
  className="Footer"
>
  <p>
    epam's ReactJS course
  </p>
</footer>
`;
